### PR TITLE
List meeting registrants

### PR DESCRIPTION
**Pull request by DanieleMancini**
List users that have registered for a meeting.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,5 @@ tmp
 build_and_publish
 .rubocop*
 .DS_Store
-.ruby-version
\ No newline at end of file
+.ruby-version
+.idea
```
Comment (kyleboe) [inline .gitignore:21]:
```suggestion
```
I don't think we need to add this to the `.gitignore`

**Review by kyleboe — CHANGES_REQUESTED** [review]
Just one small change then I'll get this merged in.

Thanks for the PR and happy [Hacktoberfest](https://hacktoberfest.com/)!

File: .gitignore (hunk 1 above)
Comment (DanieleMancini) [reply .gitignore]: done.

**Review by kyleboe — APPROVED** [review]
LGTM :+1:  

Thanks again!

**Pull request merged**